platform/barefoot/sonic-platform-modules-imt/debian/copyright

```diff
@@ -0,0 +1,13 @@
+This program is free software; you can redistribute it and/or
```

Comment:
why not our license header?

Reply:
Not sure we can use our proprietary header in a public repo